src/main/java/roomescape/login/LoginMemberArgumentResolver.java

```diff
@@ -0,0 +1,30 @@
+package roomescape.login;
+
+import org.springframework.core.MethodParameter;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+import roomescape.member.LoginMember;
+import roomescape.member.Member;
+import roomescape.member.MemberService;
+
+public class LoginMemberArgumentResolver implements HandlerMethodArgumentResolver {
+    private final MemberService memberService;
+
+    public LoginMemberArgumentResolver(MemberService memberService) {
+        this.memberService = memberService;
+    }
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.getParameterType().equals(LoginMember.class);
+    }
+
+    @Override
+    public Object resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) {
+        String token = webRequest.getHeader("Cookie").split("token=")[1];
```

Comment:
[제안 🗣️ ]
"Cookie" 같은 헤더 네임들은 상수로도 선언되어 있으니 HttpHeaders를 적극 활용해보아도 좋을 것 같아요


```suggestion
        String token = webRequest.getHeader(HttpHeaders.COOKIE).split("token=")[1];
```

Reply:
확인 했습니다 👍